src/Converter/convertToTile.js

```diff
@@ -41,12 +41,14 @@ export default {
         const builder = layer.builder;
         const parent = requester;
         const level = (parent !== undefined) ? (parent.level + 1) : 0;
+        layer.segments = layer.segments || 16;
```

Comment:
This code has a side-effect on the caller. Suggestion:

```suggestion
        const segments = layer.segments || 16;
```